docs/user-guide/connectors/mattermost.rst

```diff
@@ -44,6 +44,9 @@ run script, e.g. using:
 
 you need to supply a ``credentials.yml`` with the following content:
 
+**Remember the user value is the actual username of your bot user, not displayname**
```

Comment:
I'd rather put this into the yaml
```
  mattermost:
     url: "https://chat.example.com/api/v4"
     team: "community"
     user: "user@user.com"  #  actual username of your bot user, not displayname
     pw: "password"
     webhook_url: "https://server.example.com/webhooks/mattermost/webhook"
```

Reply:
@btotharye what do you think?

Reply:
yea agreed its a better way I'll update it

Reply:
ok @tmbo this should be updated now let me know what you think